src/frontend/apps/desk/src/features/teams/components/TeamInfo.tsx

```diff
@@ -32,58 +35,79 @@ export const TeamInfo = ({ team }: TeamInfoProps) => {
     .toLocaleString(format);
 
   return (
-    <Card className="m-b" style={{ paddingBottom: 0 }}>
-      <Box className="m-b" $direction="row" $align="center" $gap="1.5rem">
-        <IconGroup
-          width={44}
-          color={colorsTokens()['primary-text']}
-          aria-label={t('icon group')}
-          style={{
-            flexShrink: 0,
-            alignSelf: 'start',
-          }}
-        />
-        <Box>
-          <Text as="h3" $weight="bold" $size="1.25rem" className="mt-0">
-            {t('Members of “{{teamName}}“', {
-              teamName: team.name,
-            })}
-          </Text>
-          <Text $size="m">
-            {t('Add people to the “{{teamName}}“ group.', {
-              teamName: team.name,
-            })}
-          </Text>
+    <>
+      <Card className="m-b" style={{ paddingBottom: 0 }}>
+        <Box $css="align-self: flex-end;" className="m-t" $position="absolute">
+          <BoxButton
+            onClick={() => {
+              setIsModalUpdateOpen(true);
+            }}
+          >
+            <IconOptions
+              isOpen={isModalUpdateOpen}
+              aria-label={t('Open the team options modal')}
+            />
+          </BoxButton>
```

Comment:
Will you introduce the dropbutton latter on?

Reply:
Ah yes, didn't see it on the mockup.
I will add it on the PR that delete the team so.